examples/runfiles_fuzz_test.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A fuzz target that exits if it doesn't find a declared runfile.
+
+#include <cstddef>
+#include <cstdint>
+
+#include <fstream>
+#include <memory>
+#include <string>
+
+#include "tools/cpp/runfiles/runfiles.h"
+
+using bazel::tools::cpp::runfiles::Runfiles;
+
+namespace {
+  std::unique_ptr<Runfiles> runfiles = nullptr;
```

Comment:
We follow Google's C++ style guide, which does not allow non-trivially destructible types for global variables: https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

Here, given the static storage of this value, it is sufficient to store it as a global static pointer (it won't be a memory leak).